### ISSUE TITLE
UI blocked and cannot recover

### ISSUE THREAD

**Issue by Enigma-li** [AutodeskAILab/Fusion360GalleryDataset#60]
Hi,

I started the Fusion360 Gym server as suggested in the interface:

   Running
    Open Fusion 360
    Go to Tools tab > Add-ins > Scripts and Add-ins
    In the popup, select the Add-in panel, click the green '+' icon and select the server directory in this repo
    Click 'Run' to start the server

But the Fusion360 is blocked since then, even I closed and restarted. I searched so many to find a solution and even reinstalled it several times, it cannot return back. And since the UI was blocked, nothing I can do in the interface. Finally, I use the provided python tool to detach the serve, then it goes back normally.

So I suggest that add a few lines around the 'Running' command to tell users that if this case happens, try the python tool to stop the server. I think that would be helpful especially for the novice user of Fusion 360.

**Comment by karldd** [comment]
Yes I agree this is confusing. I have updated the readme with a note.
I'll also leave this open as it would be good to find a way to better handle this in the future.
